src/mmw/js/src/modeling/controllers.js

```diff
@@ -94,24 +94,30 @@ var ModelingController = {
                 } else {
                     initViews(project);
                     project.fetchResultsIfNeeded();
-                    router.navigate(project.getReferenceUrl());
+                    updateUrl();
                 }
             } else {
-                project = new models.ProjectModel({
-                    name: 'Untitled Project',
-                    created_at: Date.now(),
-                    area_of_interest: App.map.get('areaOfInterest'),
-                    scenarios: new models.ScenariosCollection()
-                });
-
-                // TODO evalutate if we can remove this global by reworking this
-                // code.
-                App.currProject = project;
-                setupNewProjectScenarios(project);
-                project.on('change:id', function() {
-                    router.navigate(project.getReferenceUrl());
-                });
+                if (!App.currProject) {
```

Comment:
Looks like removing this and the null setting src/mmw/js/src/analyze/controllers.js puts activity mode back into effect and doesn't cause any major problems. It does cause new projects to be created when jumping back and forth to compare and modeling views. But I'd suggest that maybe we make the guest behaviour a lot more like activity mode in that you get a clean project and just keep adjusting the same one rather that starting over each time  you hit the modeling page.